pandas/tests/frame/indexing/test_indexing.py

```diff
@@ -1429,7 +1429,10 @@ def test_loc_setitem_datetime_coercion(self):
         assert Timestamp("2008-08-08") == df.loc[0, "c"]
         assert Timestamp("2008-08-08") == df.loc[1, "c"]
         df.loc[2, "c"] = date(2005, 5, 5)
-        assert Timestamp("2005-05-05") == df.loc[2, "c"]
+        with tm.assert_produces_warning(FutureWarning):
+            # Comparing Timestamp to date obj is deprecated
+            assert Timestamp("2005-05-05") == df.loc[2, "c"]
```

Comment:
You can actually remove this line here, I think

Reply:
yeah @jbrockmendel makes sense

Reply:
we still want to do the assertion here though dont we?

Reply:
oh i c these are actually 2 different lines, ok leave them.

Reply:
> we still want to do the assertion here though dont we?

But on this line you are only asserting that the value at `df.loc[2, "c"]` evaluates equal (with a warning) to that timestamp, and you are already asserting that equality on the line below as well. And the fact that it raises warning is already tested elsewhere.

Reply:
1423 is asserts `Timestamp("2005-05-05") ==` while 1435 asserts `Timestamp("2005-05-05").date() ==`